src/lib/middleware/local_assets.coffee

```diff
@@ -0,0 +1,32 @@
+#
+# Serves client-side assets from Force, or from CDN
+#
+
+{ CDN_URL } = require '../../config'
+fs = require 'fs'
+path = require 'path'
+
+manifestPath = path.join(__dirname, '..', '..', '..', 'public', 'assets', 'manifest.json')
```

Comment:
In JS can use absolute paths via babel:
```javascript
import AssetMap from 'public/assets/manifest.json'
```
and check the variable for content rather than using `fs`